idea/testData/inspectionsLocal/redundantReturnLabel/inLambda.kt

```diff
@@ -0,0 +1,9 @@
+// PROBLEM: none
+fun foo(f: (String?) -> Int) {}
+
+fun test() {
+    foo {
+        if (it != null) return@foo<caret> 1
```

Comment:
Please make sure that the inspection is not suggested when using the function expression syntax:
```
foo(fun(it: String?): Int {
    if (it != null) return@foo 1
    return 0
})
```

Reply:
Why we should not suggest?

Reply:
Because this is valid code, unlike in the issue fixed in this pull request